packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -47,7 +62,7 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
   const isEnabled = (option: boolean | undefined): boolean => !(option === false);
 
   // Reduce the controls shown when mobile view is enabled.
-  if (props.mobileView) {
+  if (props.mobileView && typeof callControlsOptions !== 'boolean') {
```

Comment:
It think it would be preferable if `reduceCallControlsForMobile` can correctly handle boolean options as well.